src/devtools/views/actionList/actionList.css

```diff
@@ -24,20 +31,67 @@
 .actions {
   align-items: center;
   display: flex;
-  flex-direction: column;
-  height: 100%;
+  flex-direction: row;
   justify-content: space-between;
 }
 
 .action {
+  background: 50% 50% no-repeat;
+  background-size: 18px 18px;
+  border-radius: 50%;
+  border: 1px solid transparent;
   cursor: pointer;
   font-size: 12px;
-  height: 32%;
-  margin: 0 10px;
+  padding: 0;
   text-align: center;
   width: 100%;
+  height: 40px;
+  width: 40px;
+  opacity: 0.54;
+  transition: 0.2s all;
+  position: relative;
+}
+
+.action > span {
+  background: rgba(0, 0, 0, 0.88);
+  border-radius: 3px;
+  color: rgba(255, 255, 255, 0.88);
+  left: 50%;
+  opacity: 0;
+  padding: 4px 8px;
+  position: absolute;
+  top: -18px;
+  transform: translate(-50%, 0) scaleY(0) translateZ(0);
+  transition: 0.3s all;
 }
 
 .action:hover {
-  background-color: #423f65;
+  border-color: rgba(255, 255, 255, 0.1);
+  opacity: 0.88
+}
+
+.action:hover > span {
+  opacity: 1;
+  transform: translate(-50%, 0) translateZ(0);
+  transition-delay: 750ms;
+}
+
+.checkout {
+  composes: action;
```

Comment:
Composes is a special `css-modules` thing. It's how you reuse code. You can also do stuff like:

```css
composes: something from ".../wherever.css";
```